bin/resources/GameIndex.yaml

```diff
@@ -41058,6 +41085,9 @@ SLUS-20867:
 SLUS-20868:
   name: "MVP Baseball 2004"
   region: "NTSC-U"
+  memcardFilters:
+    - "PSCD10088" # Enables EA Sports .
```

Comment:
You deleted the word BIO :P When you put it back in make sure you don't leave an extra space in there, then I think we're golden.